math/papilo/Portfile

```diff
@@ -71,11 +74,4 @@ if {${name} eq ${subport}} {
 configure.args-append   -DGMP=ON \
                         -DQUADMATH=OFF
```

Comment:
please add:
```
-DBoost_LIBRARY_DIR_RELEASE=[boost::lib_dir] \
-DBoost_LIBRARY_DIR_DEBUG=[boost::lib_dir]
```

and I think it should all work without setting `conflicts_build`

Reply:
@barracuda156 I am pretty sure adding these, slightly unconventional, argument that upstream uses will fix the build without the need for a `conflicts_build`. Can you please verify?

Reply:
I can try tomorrow, but this is again a per-port hack, which does not address the cause of these problems.
For the record, I have opened a ticket to remove `boost`: https://trac.macports.org/ticket/71318